db/fixtures/miq_shortcuts.yml

```diff
@@ -422,9 +422,9 @@
 - :name: miq_event_definition
   :description: Control / Events
   :url: /miq_event_definition/show_list
-  :rbac_feature_name: miq_event
+  :rbac_feature_name: miq_event_definition_show_list
```

Comment:
Note, this is fixing a rename that occured but missed the feature name: https://github.com/ManageIQ/manageiq/commit/a9ff94017cf39090a6ade8d82b8f9f6a424244e2#

Reply:
I need to see if we need a migration for this change.  I don't know if this happened between releases or not.  It looks like this was within the same release:

```
joerafaniello@Joes-MBP-2 manageiq % git diff upstream/lasker upstream/master db/fixtures/miq_product_features.yml | grep -E "[\+-].+identifier" | grep event
## NOTHING
```

Jansa to lasker, it was changed and there was no migration to fix any user_roles that had that feature specifically:

```
joerafaniello@Joes-MBP-2 manageiq % git diff upstream/jansa upstream/lasker db/fixtures/miq_product_features.yml | grep -E "[\+-].+identifier" | grep event
-        :identifier: policy_edit_events
+        :identifier: miq_policy_events_assignment
+        :identifier: miq_policy_event_edit
-    :identifier: event
+    :identifier: miq_event_definition
-      :identifier: event_admin
-        :identifier: event_edit
+      :identifier: miq_event_definition_show_list
+      :identifier: miq_event_definition_show
```

I guess it's too late to fix this?  Anyone who seeded their features in lasker will have had the feature destroyed and any user_roles associated would have lost that specific feature.

Reply:
The original rename happened here: https://github.com/ManageIQ/manageiq/commit/524305676339d70217c300f953b0c0cd5af003a8

Reply:
I think this PR is fine.  We didn't do the rename in this PR.  We'll have to determine if it's a big deal that we renamed identifiers and didn't migrate any user roles that had that feature.  It's really only relevant for features where some child nodes are unselected since if a whole tree and descendants are selected, only the parent of the selected nodes is stored in the list of features in the user role.

Reply:
I'll write a migration to fix all user roles with features that were renamed for lasker in the event someone goes from pre-lasker to lasker.  That's the best we can do.